rllib/evaluation/rollout_worker.py

```diff
@@ -761,7 +759,7 @@ def wrap(env):
             else:
                 raise ValueError(
                     f"Unknown off_policy_estimation type: {method_type}! Must be "
-                    "either `simulation|is|wis|dm|dr` or a sub-class of ray.rllib."
+                    "either a class path or a sub-class of ray.rllib."
```

Comment:
Nice. Better to force explicitness and not allow too many shortcut options for the user.

Reply:
I still have the class path as a TODO, will fix that after the Trainer PR, but agree